microsoft/PowerToys
# [ImageResizer] Minimum size width and height

## Summary of the Pull Request

**What is this about:**
- Fix 1x1px image with custom size have with or height set to empty

**What is include in the PR:** 

**How does someone test / validate:** 
- Validate minimum width and height for ImageResizer sizes in settings and ImageResizer UI
- Resize an image with width or height set to 0
- 0 should be interpreted as auto


## Quality Checklist

- [x] **Linked issue:** #11116

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
